tests/org.jboss.tools.fuse.ui.bot.test/src/org/jboss/tools/fuse/ui/bot/test/utils/ProjectFactory.java

```diff
@@ -93,6 +97,29 @@ public static void createProject(String name, String archetype) throws FuseArche
 	}
 
 	/**
+	 * Removes all projects from file system.
+	 */
+	public static void deleteAllProjects() {
```

Comment:
Are you sure that this implementation always works? I'm afraid that the following part
```java
new DefaultShell("Delete Resources");
```
sometimes doesn't work since sometimes there are two shells with the same title.